scalyr_agent/builtin_monitors/tests/kubernetes_monitor_test.py

```diff
@@ -126,4 +128,40 @@ def start_test(m2, m1):
 
                 manager.stop_manager(wait_on_join=False)
                 fake_clock.advance_time(increment_by=manager_poll_interval)
-            start_test()
\ No newline at end of file
+            start_test()
+
+
+class TestExtraServerAttributes(CopyingManagerInitializationTest):
+
```

Comment:
I think we should have a test for when two monitors define the same extra server attribute key with a different value.  In the current code, the 2nd monitor will overwrite the value of the 1st (see comments below).